autonomous_software_pkg/src/controller.py

```diff
@@ -126,6 +126,12 @@ def __init__(self, gui_flag):
                 self.callback_current_state,
             )
 
+            rospy.Subscriber(
+                '/teleop_speed',
+                Int32,
+                self.callback_speed_control,
```

Comment:
The name of the callback function could be made more explicit.

Reply:
Good point, thank you. 

Renamed to callback_gui_control_throttle().